integration_tests/test_env_vars.bash

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export THAPI_HOME=${THAPI_HOME:-${PWD}}
+export THAPI_INSTALL_DIR=${THAPI_INSTALL_DIR:-${PWD}/build/ici}
```

Comment:
- Why we need `THAPI_INSTALL_DIR` ?

It seem to be used onky for `THAPI_BIN_DIR`, and was the only variable set before.
So I suggest to remove it.

Reply:
You are correct -- we don't seem to use it anywhere.